play/wikirecent/app/Dockerfile

```diff
@@ -10,13 +10,7 @@
 # TODO: Use slim once we can pip-install
```

Comment:
Is this TODO resolvable too now?

Reply:
Oh! Perhaps. Let me try now!

Reply:
Sadly we cannot! Looks like `git` is not installed in `slim`:

```
  ERROR: Error [Errno 2] No such file or directory: 'git' while executing command git clone -q https://github.com/psycopg/psycopg3.git /tmp/pip-req-build-jg6z3pea
ERROR: Cannot find command 'git' - do you have 'git' installed and in your PATH?
WARNING: You are using pip version 20.3.3; however, version 21.0 is available.
You should consider upgrading via the '/usr/local/bin/python -m pip install --upgrade pip' command.
```